router.js

```diff
@@ -45,9 +45,9 @@ router.get("/load", async (ctx) => {
   d = d.replace("dataPlaneUrl", dataPlaneUrl);
   d = d.replace("configBackendUrl", configUrl);
 
-  deviceModeInit = deviceModeInit.replaceAll("dataplaneUrl", dataPlaneUrl);
-  deviceModeInit = deviceModeInit.replaceAll("writeKey_placeHolder", writeKey);
-  deviceModeInit = deviceModeInit.replaceAll("configUrl_placeholder", configUrl);
+  deviceModeInit = deviceModeInit.replace(/dataplaneUrl/g, dataPlaneUrl);
+  deviceModeInit = deviceModeInit.replace(/writeKey_placeHolder/g, writeKey);
+  deviceModeInit = deviceModeInit.replace(/writeKey_placeHolder/g, configUrl);
```

Comment:
Why are we overriding. `writeKey_placeHolder` with configUrl?

Reply:
Seems like a copy-paste thing. Let's get this fixed ?

Reply:
Done